selftests/checkall

```diff
@@ -80,12 +80,12 @@ results_dir_content() {
 
 [ "$SKIP_RESULTSDIR_CHECK" ] || RESULTS_DIR_CONTENT="$(ls $RESULTS_DIR 2> /dev/null)"
 if [ "$TRAVIS" == "true" ]; then
-    run_rc lint 'RESULT=$(mktemp); inspekt --exclude=.git lint &>$RESULT || { cat $RESULT; rm $RESULT; exit -1; }; rm $RESULT'
+    run_rc lint 'RESULT=$(mktemp); inspekt lint --exclude=.git,.venv &>$RESULT || { cat $RESULT; rm $RESULT; exit -1; }; rm $RESULT'
```

Comment:
would you please separate the `venv` change? It's really unrelated ti tgus cgabge,,,